run_scripts/start_bag.il

```diff
@@ -1248,15 +1253,6 @@ procedure( create_layout_helper( cv tech_file inst_list rect_list via_list pin_l
                     if( inst->params != nil then
                         ; create pcell instance
                         obj = dbCreateParamInst(cv inst_cv inst->name inst->loc inst->orient 1 inst->params)
-                        ; execute parameter callbacks
```

Comment:
For reasons yet unknown to me, calling the callbacks at right after instantiating the layout does NOT work. Basically, the layout does not get updated or gets only partially updated, if the callbacks are ran at this point.